### PR TITLE
add admin support to the quickstart

### PR DESCRIPTION
**Pull request by chetvil**
- Adding admin support to the quickstart

### CODE REVIEW

**Review by rjriel — COMMENTED** [review]
Functionally good, we should do a couple of things here though:

- [ ] Create a UI for the reports
- [ ] Update the readme to include info about this new part

File: html/admin.html [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+          result.appendChild(manualInstruction);
+        }
+
+        historyContainer.appendChild(result);
+        const button = document.getElementById('verify-button')
+        button.style.display = 'none'
+      }
+    </script>
+  </body>
+</html>
```
Comment (albedoweb) [inline html/admin.html:338]: new line

File: python/src/naive_api_client.py [hunk 2]
```diff
@@ -89,4 +89,60 @@ class VerificationRequest(TypedDict):
             self.API_URL + 'verifications/incomes/',
             json=request_data,
             headers=self.API_HEADERS,
+        ).json()
+
+    def get_employee_directory_by_token(self, access_token: str) -> Any:
+        """
+        https://docs.citadelid.com/#payroll-admin
```
Comment (albedoweb) [inline python/src/naive_api_client.py:96]: not finished docstring

File: python/src/naive_api_client.py [hunk 3]
```diff
@@ -89,4 +89,60 @@ class VerificationRequest(TypedDict):
...
+
+        return requests.post(
+            self.API_URL + 'administrators/directories/',
+            json=request_data,
+            headers=self.API_HEADERS,
+        ).json()
+
+    def request_payroll_report(self, access_token: str, start_date: str , end_date: str) -> Any:
+        """
+        https://docs.citadelid.com/#payroll-admin
```
Comment (albedoweb) [inline python/src/naive_api_client.py:114]: does it make sense to have such docstrings?

**Review by albedoweb — APPROVED** [review]

**Pull request merged**